packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts

```diff
@@ -239,6 +239,11 @@ export interface IPersonaCoinProps extends IPersonaSharedProps {
    * @defaultvalue undefined
    */
   className?: string;
+
+  /**
+   * Should the persona coin change to showing the initals instead of an image on hover
+   */
+  showInitialsOnHover?: boolean;
```

Comment:
Shouldnt this be part of coinprops now?

Reply:
It should be added to IPersonaCoinProps. I had removed it from SharedPersonaProps